pilot/cmd/pilot-agent/main.go

```diff
@@ -176,7 +176,13 @@ var (
 						}
 					}
 				}
-				pilotSAN = envoy.GetPilotSAN(pilotDomain, ns)
+
+				pilotOverride := os.Getenv("PILOT_SPIFFE_OVERRIDE")
```

Comment:
Instead of using the env variable, is it more explicit to make the Pilot SAN as an argument passed to the pilot-agent?

Reply:
An arg was my first though but there seemed to be other data being pulled as env variables so I tried to keep it consistent.  I am ok either way